src/tests/enzyme_smoke.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Tribol Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (MIT)
+
+//-----------------------------------------------------------------------------
+//
+// file: enzyme_smoke.cpp
```

Comment:
do we need this?

Reply:
It follows the format in `mfem_smoke.cpp`